swift/tools/qltest.sh

```diff
@@ -6,11 +6,12 @@ QLTEST_LOG="$CODEQL_EXTRACTOR_SWIFT_LOG_DIR"/qltest.log
 
 EXTRACTOR="$CODEQL_EXTRACTOR_SWIFT_ROOT/tools/$CODEQL_PLATFORM/extractor"
 SDK="$CODEQL_EXTRACTOR_SWIFT_ROOT/qltest/$CODEQL_PLATFORM/sdk"
+RESOURCE_DIR="$CODEQL_EXTRACTOR_SWIFT_ROOT/resource-dir"
 export CODEQL_EXTRACTOR_SWIFT_LOG_LEVELS=${CODEQL_EXTRACTOR_SWIFT_LOG_LEVELS:-out:text:no_logs,out:console:info}
-
+env | sort >> $QLTEST_LOG
```

Comment:
Minor risk: if customers run `codeql test run` on their machines or in their workflows, this would mean any credentials in their environment get persisted to the test log file right?

Reply:
Yeah, that's a good catch! This is a leftover from local debugging, I'll drop it before making PR ready for review.